src/InsolationCalc.jl

```diff
@@ -10,8 +10,8 @@ function insolation(θ::FT, d::FT, param_set::APS) where {FT <: Real}
     S0::FT = tot_solar_irrad(param_set)
     AU::FT = astro_unit()
     # set max. zenith angle to π/2, insolation should not be negative
-    if θ > FT(π)/FT(2)
-        θ = FT(π)/FT(2)
+    if θ > FT(π)/2
+        θ = FT(π)/2
     end
     # weighted irradiance (3.12)
     S = S0 * (AU / d)^2
```

Comment:
I think AU here and throughout should be some parameter semimajor_axis_orbit, to be defined in ClimaParameters, so that if we change the mean planet-sun distance, we get the right behavior.

Reply:
Thanks for catching this. It is already in CLIMAParameters! I just wasn't using it here. `orbit_semimaj` defined as `1*astro_unit()` for `AbstractEarthParameterSet`